Update Diagram

Metanyu opened the pull request:
* What changed:
- Updated sequence diagram for add task feature with standardised coloring. Added activation bars for all classes for consistency.
- Updated class diagram for Task and Command classes with standardised coloring. Fixed formatting for less clustered view.

Commit:
Update Diagram

* What changed:
- Updated sequence diagram for add task feature with standardised coloring. Added activation bars for all classes for consistency.
- Updated class diagram for Task and Command classes with standardised coloring. Fixed formatting for less clustered view.

lowjunchen reviewed (approved):
good job editing the class diagram

Commit:
git checkout -b Metanyu-docs/Diagram-Update master merge branch 'Metanyu-docs/Diagram-Update'
	Update sequence diagram according to TA's feedback add colons.
	Additional activation bars for all classes for consistency.
	Changed coloring to meet standard.

	Update class diagram for Task and Command classes with
	standardised coloring. Fixed formatting for less clustered view.

Commit:
Resolve conflicts by taking newer version from fork

Metanyu committed:
Merge pull request #3 from Metanyu/master

Resolve conflicts
